--- a/build-win64-mxe/build/vips-web.mk
+++ b/build-win64-mxe/build/vips-web.mk
@@ -8,7 +8,7 @@
 $(PKG)_GH_CONF  := libvips/libvips/releases,v,a,,,.tar.xz
 $(PKG)_SUBDIR   := vips-$($(PKG)_VERSION)
 $(PKG)_FILE     := vips-$($(PKG)_VERSION).tar.xz
-$(PKG)_DEPS     := cc meson-wrapper libwebp librsvg glib pango libarchive \
+$(PKG)_DEPS     := cc meson-wrapper libwebp glib libarchive \
                    libjpeg-turbo tiff lcms libexif libheif libpng \
                    libspng libimagequant highway cgif
 
@@ -17,19 +17,17 @@
     mkdir -p $(PREFIX)/$(TARGET)/vips-packaging
     $(foreach f, ChangeLog LICENSE README.md, \
         cp '$(SOURCE_DIR)/$(f)' '$(PREFIX)/$(TARGET)/vips-packaging';)
+    cp '/data/THIRD-PARTY-NOTICES' '$(PREFIX)/$(TARGET)/vips-packaging'
 
     (printf '{\n'; \
      printf '  "aom": "$(aom_VERSION)",\n'; \
      printf '  "archive": "$(libarchive_VERSION)",\n'; \
-     printf '  "cairo": "$(cairo_VERSION)",\n'; \
      printf '  "cgif": "$(cgif_VERSION)",\n'; \
      printf '  "exif": "$(libexif_VERSION)",\n'; \
      printf '  "expat": "$(expat_VERSION)",\n'; \
      printf '  "ffi": "$(libffi_VERSION)",\n'; \
-     printf '  "fontconfig": "$(fontconfig_VERSION)",\n'; \
      printf '  "freetype": "$(freetype_VERSION)",\n'; \
      printf '  "fribidi": "$(fribidi_VERSION)",\n'; \
-     printf '  "gdkpixbuf": "$(gdk-pixbuf_VERSION)",\n'; \
      $(if $(IS_INTL_DUMMY),,printf '  "gettext": "$(gettext_VERSION)"$(comma)\n';) \
      printf '  "glib": "$(glib_VERSION)",\n'; \
      printf '  "harfbuzz": "$(harfbuzz_VERSION)",\n'; \
@@ -41,11 +39,8 @@
           $(if $(IS_MOZJPEG),,printf '  "jpeg": "$(libjpeg-turbo_VERSION)"$(comma)\n';)) \
      printf '  "lcms": "$(lcms_VERSION)",\n'; \
      $(if $(IS_MOZJPEG),printf '  "mozjpeg": "$(mozjpeg_VERSION)"$(comma)\n';) \
-     printf '  "pango": "$(pango_VERSION)",\n'; \
-     printf '  "pixman": "$(pixman_VERSION)",\n'; \
      printf '  "png": "$(libpng_VERSION)",\n'; \
      $(if $(IS_INTL_DUMMY),printf '  "proxy-libintl": "$(proxy-libintl_VERSION)"$(comma)\n';) \
-     printf '  "rsvg": "$(librsvg_VERSION)",\n'; \
      printf '  "spng": "$(libspng_VERSION)",\n'; \
      printf '  "tiff": "$(tiff_VERSION)",\n'; \
      printf '  "vips": "$(vips-web_VERSION)",\n'; \
@@ -72,6 +67,9 @@
         -Dexamples=false \
         -Dintrospection=disabled \
         -Dmodules=disabled \
+        -Dfontconfig=disabled \
+        -Dpangocairo=disabled \
+        -Dpixbuf=disabled \
         -Dcfitsio=disabled \
         -Dfftw=disabled \
         -Djpeg-xl=disabled \
