--- a/build-win64-mxe/build/plugins/all-deps/vips-all.mk
+++ b/build-win64-mxe/build/plugins/all-deps/vips-all.mk
@@ -8,14 +8,14 @@
 $(PKG)_SUBDIR    = $(vips_SUBDIR)
 $(PKG)_FILE      = $(vips_FILE)
 $(PKG)_URL       = $(vips_URL)
-$(PKG)_DEPS     := $(vips_DEPS) imagemagick matio openexr cfitsio \
-                   nifticlib poppler fftw openslide libjxl libraw
+$(PKG)_DEPS     := $(vips_DEPS) fftw openslide

 define $(PKG)_PRE_CONFIGURE
     # Copy some files to the packaging directory
     mkdir -p '$(PREFIX)/$(TARGET)/vips-packaging'
     $(foreach f, ChangeLog LICENSE README.md, \
         cp '$(SOURCE_DIR)/$(f)' '$(PREFIX)/$(TARGET)/vips-packaging';)
+    cp '/data/THIRD-PARTY-NOTICES' '$(PREFIX)/$(TARGET)/vips-packaging'

     (printf '{\n'; \
      printf '  "aom": "$(aom_VERSION)",\n'; \
