--- a/build-win64-mxe/build/plugins/all-deps/overrides.mk
+++ b/build-win64-mxe/build/plugins/all-deps/overrides.mk
@@ -10,10 +10,19 @@

 vips_MESON_OPTS = \
     -Dmodules=enabled \
-    -Dheif-module=$(if $(IS_HEVC),enabled,disabled) \
-    $(if $(findstring graphicsmagick,$(vips_all_DEPS)), -Dmagick-package=GraphicsMagick) \
+    -Dfontconfig=disabled \
+    -Dpangocairo=disabled \
+    -Dheif=disabled \
+    -Djpeg-xl=disabled \
+    -Dmagick=disabled \
+    -Dmatio=disabled \
+    -Dnifti=disabled \
+    -Dopenexr=disabled \
+    -Dpoppler=disabled \
     -Dpdfium=disabled \
     -Dquantizr=disabled \
+    -Duhdr=disabled \
+    -Draw=disabled \
     -Dspng=disabled

 # https://github.com/libvips/build-win64-mxe/issues/80
